### PR TITLE
Add rubygems keys to push from travis

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
Bring https://github.com/mdsol/kender/pull/71 up to date

@jfeltesse-mdsol @ykitamura-mdsol @ssteeg-mdsol

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,15 @@ language: ruby
 
 rvm:
   - 2.3.7
-  - 2.4.4
```
Comment (jfeltesse-mdsol) [inline .travis.yml]: [nits] latest 2.3 is `2.3.8`. doesn't matter so much but just FYI I guess.

Comment (jcarres-mdsol) [reply .travis.yml]: good, done.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+  - 2.6.1
 
 script: bundle exec cucumber
+
+deploy:
+  provider: rubygems
+  api_key:
+    secure: QZG6pQmgoKlE4TI7SkCp87HdUL1s+47YpQiCldkG23Xef4jVkEk1GzsFLJnaBhOoExPDhRhog/rEN+pzcSNmx+3W0AZwy/Z5YQ2VNn9ojd8hhCTx4uXO1iagMXfavZvF+olzHJ6PRD9MzqHIm+MU1xFI8x5QqDrGY4XVSK/DyQw=
+  on:
+    tags: true
```
Comment (ykitamura-mdsol) [inline .travis.yml]: I think it's safer to have `condition`:
https://github.com/openzipkin/zipkin-ruby/blob/master/.travis.yml#L18

Comment (jcarres-mdsol) [reply .travis.yml]: done

**Pull request merged**